gui/wxpython/mapdisp/frame.py

```diff
@@ -142,7 +142,37 @@ def __init__(
         #
         self.statusbarManager = None
         if statusbar:
-            self.statusbar = self.CreateStatusbar()
+            # items for choice
+            statusbarItems = [
+                sb.SbCoordinates,
+                sb.SbRegionExtent,
+                sb.SbCompRegionExtent,
+                sb.SbShowRegion,
+                sb.SbAlignExtent,
+                sb.SbResolution,
+                sb.SbDisplayGeometry,
+                sb.SbMapScale,
+                sb.SbGoTo,
+                sb.SbProjection,
+            ]
+            self.statusbarItemsHiddenInNviz = (
+                sb.SbAlignExtent,
+                sb.SbDisplayGeometry,
+                sb.SbShowRegion,
+                sb.SbResolution,
+                sb.SbMapScale,
+            )
+            self.statusbar = self.CreateStatusbar(statusbarItems)
+
+            self.Map.GetRenderMgr().updateProgress.connect(
+                self.statusbarManager.SetProgress
+            )
+            self.Map.GetRenderMgr().renderingFailed.connect(
+                lambda cmd, error: self._giface.WriteError(
+                    _("Failed to run command '%(command)s'. Details:\n%(error)s")
+                    % dict(command=" ".join(cmd), error=error)
+                )
+            )
```

Comment:
Didn't notice this before, but this doesn't seem to have anything to do with statusbar, so it shouldn't be inside the if.